src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -2,11 +2,15 @@
   "dependencies": {
     "System.Runtime": "4.1.1-beta-24403-05",
     "Microsoft.NETCore.Platforms": "1.0.2-beta-24403-05",
-    "System.Console": "4.0.1-beta-24403-05",
+    "System.Console": "4.0.0",
```

Comment:
This is due to a conflict with a higher version (System.Console 4.1.0) that test projects use that reference the RemoteExecutor project.